task/project/task/serializers.py

```diff
@@ -1,10 +1,34 @@
 from rest_framework import serializers
-
 from .models import TaskModel
 
 
-class CreateTaskSerializers(serializers.ModelSerializer):
+class TaskSerializers(serializers.ModelSerializer):
 
     class Meta:
         model = TaskModel
         fields = "__all__"
+
+
+class TaskDetailsSerializer(serializers.ModelSerializer):
+    assignee = serializers.SerializerMethodField()
+    reporter = serializers.SerializerMethodField()
+
+    class Meta:
+        model = TaskModel
+        fields = ("task", "assignee","reporter")
+
+    def get_assignee(self, obj):
+        user = {
```

Comment:
it's a kind of duplicate code as we are using the same response for the user and reporter.

Reply:
And each time for every single field we are fetching from 'obj.{user}.{profile}.e=xyz field.'